pipeline_api/src/main/java/com/splicemachine/pipeline/writehandler/UpdateUtils.java

```diff
@@ -0,0 +1,84 @@
+package com.splicemachine.pipeline.writehandler;
+
+import com.carrotsearch.hppc.BitSet;
+import com.carrotsearch.hppc.BitSetIterator;
+import com.splicemachine.encoding.MultiFieldDecoder;
+import com.splicemachine.kvpair.KVPair;
+import com.splicemachine.storage.EntryDecoder;
+import com.splicemachine.storage.index.BitIndex;
+import com.splicemachine.storage.index.BitIndexing;
+import com.splicemachine.utils.ByteSlice;
+
```

Comment:
or add a description here. PR descriptions are good but we should also copy the necessary information to understand this code over here.

Reply:
Added explanation